src/lib/features/change-request-access-service/sql-change-request-access-read-model.ts

```diff
@@ -0,0 +1,52 @@
+import { SKIP_CHANGE_REQUEST } from '../../types';
+import { Db } from '../../db/db';
+import { AccessService } from '../../services';
+import User from '../../types/user';
+import { IChangeRequestAccessReadModel } from './change-request-access-read-model';
+
+export class ChangeRequestAccessReadModel
+    implements IChangeRequestAccessReadModel
+{
+    private db: Db;
+
+    private accessService: AccessService;
+
+    constructor(db: Db, accessService: AccessService) {
+        this.db = db;
+        this.accessService = accessService;
+    }
+
+    public async canBypassChangeRequest(
+        project: string,
+        environment: string,
+        user?: User,
+    ): Promise<boolean> {
+        const [canSkipChangeRequest, changeRequestEnabled] = await Promise.all([
+            user
+                ? this.accessService.hasPermission(
```

Comment:
this should be another read model because to check permissions I don't need all commands on the access service. But for now let's leave this as another refactor